lib/nvmf/nvmf_internal.h

```diff
@@ -178,6 +178,14 @@ struct spdk_nvmf_ns {
 	bool zcopy;
 	/* Command Set Identifier */
 	enum spdk_nvme_csi csi;
+	/* Namespace state reflects the subsystem's one; in addition a namespace can
+	 * be paused if the subsystem is active. Paused namespace means that only I/O
+	 * is stopped, keep alive for example goes on because the subsystem is active.
+	 * So namespace paused state can be considered as a subset of the subsystem's
+	 * paused state that doesn't need a new status in spdk_nvmf_subsystem_state
+	 * to cooperate with other management operations.
```

Comment:
How many kinds of states does ns_state contain? It seems that it does not contain intermediate states like `SPDK_NVMF_SUBSYSTEM_DEACTIVATING`, `SPDK_NVMF_SUBSYSTEM_RESUMING`, `SPDK_NVMF_SUBSYSTEM_ACTIVATING`, `SPDK_NVMF_SUBSYSTEM_PAUSING`...
If ns_state has intermediate states, the below `nvmf_ns_state_change` implementation should be similar to `nvmf_subsystem_state_change`